bin/check-kube-service-available.rb

```diff
@@ -68,6 +68,8 @@ def run
       a.spec.selector.to_h.each do |k, v|
         selector_key << "#{k}=#{v}"
       end
+      # If selector_key is empty skip this service
+      next unless selector_key.any?
```

Comment:
is there some reason to use `unless selector_key.any?` vs `if selector_key.empty?` I think using `empty` in this case makes it more obvious and makes it so that we don't need a comment above because the code is more straightforward. I may very well be missing some nuance though.

Reply:
I am new to Ruby :) and since unless worked for me I left it in the code. However, no problem with the if statement. I updated the code already per your comment.